guide/daily-spending-wallet/security.md

```diff
@@ -362,6 +371,18 @@ By including UX patterns for hiding information pattern wallets, we give users a
 
 ---
 
+## Preventing theft when offline
+
+Mobile wallets may frequently be offline, either due to a poor connection or because the application is not running. To improve battery live, mobile operating systems tend to prevent applications the user is not actively using from running code in the background. This can prevent mobile wallets from being reliable monitors of its payment channels. Malicious actors who share channels with the user may take advantage of these moments to try to steal funds by closing channels with incorrect data.
+
+A simple technique to reduce this risk is to notify the user when the application has been offline for an extended period of time. [Forced channel closes](https://wiki.ion.radar.tech/tech/channels/channel-closing#force-close) have an agreed upon delay built-in. Notifications should be sent early enough so that users have time to appropriately respond.
```

Comment:
```suggestion
A technique to reduce this risk is to notify the user when the application has been offline for an extended period of time to open the app so their channels can be checked. [Forced channel closes](https://wiki.ion.radar.tech/tech/channels/channel-closing#force-close) have an agreed upon delay built-in. Notifications should be sent earlier than this delay so that users have time to appropriately respond.
```